core/db_models/EEM_Form_Input.model.php

```diff
@@ -188,19 +188,31 @@ protected function __construct(Element $element, InputTypes $input_types, $timez
                 ),
                 'FIN_status'    => new EE_Enum_Text_Field(
                     'FIN_status',
-                    esc_html__(
-                        'Whether form input is active, archived, trashed, or used as a default on new forms. Values correspond to the EEM_Form_Input::STATUS_* constants.',
-                        'event_espresso'
+                    esc_html(
+                        sprintf(
+                            /* translators: 1 class name */
+                            __(
+                                'Whether form input is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
+                                'event_espresso'
+                            ),
+                            'EEM_Form_Input::STATUS_*'
+                        )
```

Comment:
why split out the escaping?

does this not do the same?

```suggestion
                        sprintf(
                            /* translators: 1 class name */
                            esc_html__(
                                'Whether form input is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
                                'event_espresso'
                            ),
                            'EEM_Form_Input::STATUS_*'
                        )
```

Reply:
I thought it is better to use escaping at the end. If you prefer the other way, I am totally fine with that.

Reply:
Also, it's possible that the placeholder text may have some special characters sometimes, so escaping that will be skipped if we escape early.

Reply:
ya I'm fine with either way too. I just wanted to know if there was some deeper reason for doing it this way that I was unaware of. 

i don't think it's necessary in this case since doing the escaping at the end would only additionally affect the variables injected into the placeholders. In this case `EEM_Form_Input::STATUS_*` will not require any escaping, but if it contained any HTML then we would likely want to do it the way you have here.

but again, I'm fine with either way, so no sense in changing it